integration/daemon/daemon_test.go

```diff
@@ -652,6 +652,58 @@ func testLiveRestoreVolumeReferences(t *testing.T) {
 		assert.NilError(t, err)
 	})
 
+	t.Run("image mount", func(t *testing.T) {
```

Comment:
Seems like this can run  with `t.Parallel` since its using a custom daemon.
But, I guess if we don't require the extra daemon opt like I mentioned above then that would be different.